src/e3/log.py

```diff
@@ -379,7 +381,10 @@ def activate_with_args(args: Namespace, default_level: int = logging.WARNING) ->
     if args.verbose > 0:
         level = default_level - 10 * args.verbose
     else:
-        level = args.loglevel
+        if args.loglevel in LEVELS:
```

Comment:
why do you need the if ? if an issue with mypy an assert is sufficient

Reply:
Humm, I do not understand why mypy comes into play here?